install-deps.sh

```diff
@@ -32,6 +32,7 @@ if [ x`uname`x = xFreeBSDx ]; then
         devel/gperf \
         devel/gmake \
         devel/cmake \
+        devel/cunit \
```

Comment:
@wjwithagen i notice that you don't want to test SPDK support on FreeBSD atm, so i removed this line in #18944.

@liupan1111 ^